benchmarks/time_solve_models.py

```diff
@@ -22,11 +22,328 @@ def prepare_model(model):
     disc.process_model(model)
 
 
+def prepare_model_ORegan2021(model):
+    geometry = model.default_geometry
+
+    # load parameter values and process model and geometry
+    param = pybamm.ParameterValues("ORegan2021")
```

Comment:
If you use parameters (as in my previous comment) you can have a single `prepare_model(model, parameters)` function.